commands/swapdb.md

```diff
@@ -0,0 +1,19 @@
+This new command swaps two Redis databases, so that immediately all the
+clients connected to a given DB will see the data of the other DB, and
+the other way around. Example:
```

Comment:
Please expand `DB` to `database`. Makes reading easier. Applies to below as well.